paddle/fluid/framework/paddle2cinn/cinn_cache_key.cc

```diff
@@ -39,13 +44,42 @@ CinnCacheKey::CinnCacheKey(const ir::Graph& graph,
   this->SetKey(graph, input_shapes, arch_str);
 }
 
+std::string CinnCacheKey::HashGraph(const ir::Graph& graph) {
+  // using Dot to unqiue graph
+  inference::analysis::Dot dot;
+  std::unordered_map<const ir::Node*, std::string> node2dot;
```

Comment:
Just a question, can we reuse id of `ir::Node` instead of creating your own id?

In that case we can save the `std::unordered_map` data structure and corresponding insert/find operations

Reply:
`node.h` states that DO NOT USE node id. To be discussed in the future.